actors/account/Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [package]
 name = "fil_actor_account"
 description = "Builtin account actor for Filecoin"
-version = "12.0.0"
-license = "MIT OR Apache-2.0"
+version.workspace = true
+license.workspace = true
+edition.workspace = true
+repository.workspace = true
 authors = ["ChainSafe Systems <info@chainsafe.io>", "Protocol Labs", "Filecoin Core Devs"]
```

Comment:
Can we not do the same thing for Authors?

Reply:
Not all the author lists are the same.

Reply:
That's likely often an oversight. I know you're going to point at the EVM actors as a case where it's not, but ... is it really worth distinguishing?